Heroes.SDK.Library/Definitions/Structures/Player/Physics.cs

```diff
@@ -13,87 +13,87 @@ public struct Physics
         /// <summary>
         /// Amount of frames in which the character will fall at a decreased rate when rolling off a ledge.
         /// </summary>
-        public int HangTime { get; set; }
+        public int jump2_timer { get; set; }
 
         /// <summary>
         /// Higher values indicate easier acceleration on rough surfaces.
         /// </summary>
-        public float FloorGrip { get; set; }
+        public float pos_error { get; set; }
 
         /// <summary>
         /// The horizontal speed cap, maximum speed in X/Z axis.
         /// </summary>
-        public float HorizontalSpeedCap { get; set; }
+        public float lim_h_spd { get; set; }
 
         /// <summary>
         /// The vertical speed cap, maximum speed in Y axis.
         /// </summary>
-        public float VerticalSpeedCap { get; set; }
+        public float lim_v_spd { get; set; }
 
         /// <summary>
         /// Affects character acceleration.
         /// </summary>
-        public float UnknownAccelRelated { get; set; }
+        public float max_x_spd { get; set; }
```

Comment:
Can the comments be updated here in some places? Since we know the variable names now.